### PR TITLE
Release `0.18.1`

### PR DESCRIPTION
**Pull request by ureeves**
## Added

- Add `allow_signature_message`, `stake_signature_message`,
  `unstake_signature_message`, and `withdraw_signature_message`
  to generate signature messages for stake contract interaction [#119]
- Add `stct_signature_message` and `stco_signature_message` to generate
  signature messages for transfer contract interaction [#119]
- Add `Stake`, `Unstake`, `Withdraw`, `Allow`, and `StakeData` structs to allow
  interaction with the stake contract [#119]
- Add `Stct`, `Wfct`, `Stco`, `Wfco`, `Wfctc`, `Mint`, and `TreeLeaf` structs to
  allow interaction with the transfer contract [#119]
- Add `Transaction` structure [#116]

[0.18.1]: https://github.com/dusk-network/phoenix-core/compare/v0.18.0...v0.18.1

### CODE REVIEW

**Review by fed-franz — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
```
Comment (HDauven) [inline CHANGELOG.md:8]: Don't we normally leave `Unreleased` around even if there's nothing to list?

**Review by HDauven — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (ureeves) [reply CHANGELOG.md:8]: Nice catch, will change

Comment (ureeves) [reply CHANGELOG.md:8]: Done

**Review by HDauven — APPROVED** [review]
LGTM!

**Pull request merged**